roles/iofiller/files/Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/ubi:latest
+COPY yum.repos.d/* /etc/yum.repos.d/

```

Comment:
Can we remove the repo files post the deployment of required packages?

Reply:
With volume mount, we no longer copy yum.repos.d/*